test/customForceUpdate.test.tsx

```diff
@@ -0,0 +1,131 @@
+import mockConsole from "jest-mock-console"
+import { observable, runInAction } from "mobx"
+import * as React from "react"
+import { useCallback, useState } from "react"
+import { cleanup, render } from "react-testing-library"
+import { useForceUpdate, useObserver } from "../src"
+
+afterEach(cleanup)
+
+describe("is used to make calls to force update skip re-renderings on demand", () => {
```

Comment:
Some people are getting inspiration from tests on how to use the lib and I would consider these examples a bad practice. Setting state in a render phase just doesn't feel right. Feel free to get wild in your lib, but not here, please.

Let's have a simple test for passing that custom force update and validating it works.

Reply:
done, simplified